src/components/upgrader/boxfishUpgrader.ts

```diff
@@ -444,6 +445,11 @@ export default class BoxfishUpgrader extends EventEmitter implements IDeviceUpgr
       const response = await this._cameraManager.getState();
 
       this._logger.info(`Upgrade status ${JSON.stringify(response)}`);
+      if (response.status === 10) {
+        // EMMC is not ready lets wait and try again
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Why is this necessary?

Reply:
To give the EMMC time to come up (status 10, means it is not ready yet)

Reply:
Alright. Nice